packages/axios-extension/src/abap/types/adt-types.ts

```diff
@@ -0,0 +1,30 @@
+export interface AdtSchemaData {
```

Comment:
Can we have some comments (JS Doc or otherwise) about these types please? I'm mainly focussed on what they're useful for